src/sentry/static/sentry/app/actionCreators/broadcasts.jsx

```diff
@@ -0,0 +1,11 @@
+export function getAll(api) {
```

Comment:
What about if the function name had a bit more context (e.g. `getAllBroadcasts`) so you don't have to `import * as broadcastActions`?

Reply:
Sure.